data/json/field_type.json

```diff
@@ -0,0 +1,632 @@
+[
+  {
+    "id": "fd_null",
+    "type": "field_type",
+    "legacy_enum_id": 0,
+    "intensity_levels": [ { "name": "nothing" } ]
+  },
+  {
+    "id": "fd_blood",
+    "type": "field_type",
+    "legacy_enum_id": 1,
+    "intensity_levels": [ { "name": "blood splatter", "color": "red" }, { "name": "blood stain" }, { "name": "puddle of blood" } ],
+    "half_life": "2 days",
+    "phase": "liquid",
+    "accelerated_decay": true,
+    "is_draw_field": true
+  },
+  {
+    "id": "fd_bile",
+    "type": "field_type",
+    "legacy_enum_id": 2,
+    "intensity_levels": [ { "name": "bile splatter", "color": "pink" }, { "name": "bile stain" }, { "name": "puddle of bile" } ],
+    "half_life": "1 days",
+    "phase": "liquid",
+    "accelerated_decay": true,
+    "is_draw_field": true
+  },
+  {
+    "id": "fd_gibs_flesh",
+    "type": "field_type",
+    "legacy_enum_id": 3,
+    "intensity_levels": [
+      { "name": "scraps of flesh", "sym": "~", "color": "brown" },
+      { "name": "bloody meat chunks", "color": "light_red" },
+      { "name": "heap of gore", "color": "red" }
+    ],
+    "half_life": "2 days",
+    "phase": "solid",
+    "accelerated_decay": true,
+    "is_draw_field": true
+  },
+  {
+    "id": "fd_gibs_veggy",
+    "type": "field_type",
+    "legacy_enum_id": 4,
+    "intensity_levels": [
+      { "name": "shredded leaves and twigs", "sym": "~", "color": "light_green" },
+      { "name": "shattered branches and leaves" },
+      { "name": "broken vegetation tangle", "color": "green" }
+    ],
+    "half_life": "2 days",
+    "phase": "solid",
+    "accelerated_decay": true,
+    "is_draw_field": true
+  },
+  {
+    "id": "fd_web",
+    "type": "field_type",
+    "legacy_enum_id": 5,
+    "intensity_levels": [ { "name": "cowebs", "sym": "}" }, { "name": "webs" }, { "name": "thick webs", "transparent": false } ],
+    "priority": 2,
+    "phase": "solid",
+    "do_item": false,
+    "is_draw_field": true
+  },
+  {
+    "id": "fd_slime",
+    "type": "field_type",
+    "legacy_enum_id": 6,
+    "intensity_levels": [
+      { "name": "slime trail", "color": "light_green" },
+      { "name": "slime stain" },
+      { "name": "uddle of slime", "color": "green" }
+    ],
+    "half_life": "1 days",
+    "phase": "liquid",
+    "accelerated_decay": true,
+    "is_draw_field": true
+  },
+  {
+    "id": "fd_acid",
+    "type": "field_type",
+    "legacy_enum_id": 7,
+    "intensity_levels": [
+      { "name": "acit splatter", "sym": "5", "color": "light_green", "dangerous": true },
```

Comment:
I'm not a native English speaker, is "acit" a real word or just a misspelling?

Reply:
Probably was meant to be "acid"

Reply:
Thanks! That was a typo.

![image](https://user-images.githubusercontent.com/16213433/61471629-babce280-a98b-11e9-883e-95b65c7be0b7.png)